test/unit/refresh.js

```diff
@@ -1296,6 +1296,57 @@ describe('swiftserver:refresh', function () {
 
   });
 
+  describe('Generated a bff application with alert notification for bluemix', function() {
```

Comment:
I'm not sure your test needs it, but here's an example spec from a default BFF app with no services:
```
{                                                                                                                                                                            
  "appType": "scaffold",
  "appName": "example-bff",
  "bluemix": true,
  "docker": true,
  "web": true,
  "exampleEndpoints": true,
  "hostSwagger": true,
  "services": {},
  "capabilities": {
    "metrics": true
  },
  "config": {
    "logger": "helium",
    "port": 8080
  }
}
```